src/simulators/matrix_product_state/matrix_product_state_tensor.hpp

```diff
@@ -442,6 +444,7 @@ MPS_Tensor MPS_Tensor::contract(const MPS_Tensor &left_gamma,
 //---------------------------------------------------------------
 void MPS_Tensor::contract_2_dimensions(const MPS_Tensor &left_gamma, 
 			               const MPS_Tensor &right_gamma,
+				       uint_t omp_threads,
```

Comment:
Can we remove this parameter and use: `MPS::get_omp_threads()` instead?

Reply:
I don't see how I can do this, because this file does not recognize the MPS class. I tried adding `class MPS` at the beginning, but it still doesn't work. Any suggestions?

Reply:
Ok, leave it like this